dev/io.openliberty.microprofile.telemetry.1.1.internal/transformed.bnd

```diff
@@ -0,0 +1,26 @@
+#*******************************************************************************
+# Copyright (c) 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/transform.props
+
+
+Bundle-Name: io.openliberty.microprofile.telemetry.1.1.internal.jakarta
+Bundle-SymbolicName: io.openliberty.microprofile.telemetry.1.1.internal.jakarta
+
+Export-Package: \
+	io.openliberty.microprofile.telemetry11.internal.cdi;version='1.0',\
+	io.openliberty.microprofile.telemetry11.internal.rest;version='1.0';thread-context=true,\
+	io.openliberty.microprofile.telemetry11.internal.rest.jakarta;version='1.0';thread-context=true
+	
+-dsannotations: \
+  io.openliberty.microprofile.telemetry11.internal.cdi.SPIMetaData,\
+  io.openliberty.microprofile.telemetry11.internal.rest.jakarta.TelemetryClientBuilderListener,\
+  io.openliberty.microprofile.telemetry11.internal.sdk.OpenTelemetrySdkBuilderSupplierImpl,\
+  io.openliberty.microprofile.telemetry11.internal.rest.TelemetryServletContainerInitializer
```

Comment:
Why do we have dsannotations in the transformed.bnd?

Reply:
io.openliberty.microprofile.telemetry11.internal.rest.jakarta.TelemetryClientBuilderListener is Jakarta specific as `ClientBuilderListener ` isn't supported with javax

Reply:
I don't think this actually helps though, both bundles would still import the package containing the interface and that package shouldn't be present when running in EE 7 or 8.

Reply:
In general, the transformer is strictly for having the same code which just needs the package names changed to work with jakarta vs. javax. If you actually need different code for EE8 vs. EE9 then you should have separate bundles.

Reply:
Yes but the mechanism used for transformation is sub-bundles and what is done here works as well.  The dsannotations and the exports are different between the original.bnd and transformed.bnd.  We are creating two bundles.  We are just using sub-bundles to do it.  I had this same questions when doing the initial review and then compared original.bnd and transformed.bnd and noted that they are export and doing different dsannotations between the two.

Reply:
For this to work though you are correct that we wouldn't have the class at all if it is not used by the sub-bundle.

Reply:
Yeah, on closer investigation the `io.openliberty.microprofile.telemetry11.internal.rest.jakarta` package is only included in the transformed bundle and the imports for each bundle are calculated appropriately for its contents.

Adding the package to `Export-Package` here causes it to be included in the bundle at all as well as being imported which wasn't immediately obvious.

I don't think this package actually needs to be exported or available on the thread context. If that turns out to be correct I'll change it to a private package and add comments noting that the contents of the two bundles is different.